dashboard/test/factories/factories.rb

```diff
@@ -355,7 +356,7 @@
           email: user.email,
           hashed_email: user.hashed_email,
           credential_type: AuthenticationOption::GOOGLE,
-          authentication_id: 'abcd123',
+          authentication_id: "abcd#{user.id}",
```

Comment:
A new authentication_id is assigned to a user instead of the same hard-coded id.

example of some errors cause by failed validation:: Credential type has already been taken
user_test
```
ERROR["test_find_for_authentication_finds_migrated_Google_email_user", "UserTest", 533.7600649999804]
 test_find_for_authentication_finds_migrated_Google_email_user#UserTest (533.76s)
ActiveRecord::RecordInvalid:         ActiveRecord::RecordInvalid: Validation failed: Credential type has already been taken
            test/factories/factories.rb:355:in `block (4 levels) in <top (required)>'
            test/models/user_test.rb:792:in `block in <class:UserTest>'
            test/testing/setup_all_and_teardown_all.rb:22:in `run'

ERROR["test_update_primary_contact_info_adds_new_email_option_for_student_if_no_matches_exist", "UserTest", 557.0873439999996]
 test_update_primary_contact_info_adds_new_email_option_for_student_if_no_matches_exist#UserTest (557.09s)
ActiveRecord::RecordInvalid:         ActiveRecord::RecordInvalid: Validation failed: Credential type has already been taken
            test/factories/factories.rb:355:in `block (4 levels) in <top (required)>'
            test/models/user_test.rb:1922:in `block in <class:UserTest>'
            test/testing/setup_all_and_teardown_all.rb:22:in `run'

```

admin_search_controller test
```
ERROR["test_admin_calling_get_find_students_should_receive_success", "AdminSearchControllerTest", 106.93024699995294]
 test_admin_calling_get_find_students_should_receive_success#AdminSearchControllerTest (106.93s)
ActiveRecord::RecordInvalid:         ActiveRecord::RecordInvalid: Validation failed: Credential type has already been taken
            test/factories/factories.rb:355:in `block (4 levels) in <top (required)>'
            test/test_helper.rb:464:in `block in test_user_gets_response_for'
            test/testing/setup_all_and_teardown_all.rb:22:in `run'
```